components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -141,6 +141,39 @@ def fileread_gen(fin, fsize, bufsize):
     fin.close()
 
 
+def countAnnotations(queryResult=[]):
+    """
+    Count the different the annotions from the
+    provided projection query result
+    """
+
+    counts = {
+        "TagAnnotation": 0,
+        "FileAnnotation": 0,
+        "CommentAnnotation": 0,
+        "LongAnnotation": 0,
+        "MapAnnotation": 0,
+        "OtherAnnotation": 0}
+
+    for r in queryResult:
```

Comment:
did you see the discussion about https://github.com/openmicroscopy/openmicroscopy/pull/5056 ? this code is not needed